packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -173,6 +173,71 @@ void main() {
     log.clear();
   });
 
+  testWidgetsWithLeakTracking('PaginatedDataTable footer with default rowsPerPage Title', (WidgetTester tester) async {
+    int rowsPerPage = 2;
+    String defaultRowsPerPageTitle = 'Rows per page:';
+
+    Widget buildTable(TestDataSource source, int rowsPerPage) {
+      return PaginatedDataTable(
+        header: const Text('Test table'),
+        source: source,
+        rowsPerPage: rowsPerPage,
+        showFirstLastButtons: true,
+        availableRowsPerPage: const <int>[
+          2, 3, 4, 5, 7, 8,
+        ],
+        onRowsPerPageChanged: (int? rowsPerPage) {
+        },
+        onPageChanged: (int rowIndex) {
+        },
+        columns: const <DataColumn>[
+          DataColumn(label: Text('Name')),
+          DataColumn(label: Text('Calories'), numeric: true),
+          DataColumn(label: Text('Generation')),
+        ],
+      );
+    }
+
+    await tester.pumpWidget(MaterialApp(
+      home: buildTable(source, rowsPerPage)
+    ));
+
+    expect(find.text(defaultRowsPerPageTitle), findsOneWidget);
+  });
```

Comment:
```suggestion
  testWidgetsWithLeakTracking('Default rowsPerPageTitle', (WidgetTester tester) async {
    await tester.pumpWidget(MaterialApp(
      home: PaginatedDataTable(
        header: const Text('Test table'),
        source: source,
        // This callback shows the rows page title in the footer.
        onRowsPerPageChanged: (int? rowsPerPage) { },
        rowsPerPage: 2,
        availableRowsPerPage: const <int>[
          2, 3, 4, 5, 7, 8,
        ],
        columns: const <DataColumn>[
          DataColumn(label: Text('Name')),
          DataColumn(label: Text('Calories'), numeric: true),
          DataColumn(label: Text('Generation')),
        ],
      ),
    ));

    final MaterialLocalizations localizations = MaterialLocalizations.of(
      tester.element(find.byType(PaginatedDataTable)),
    );
    expect(find.text(localizations.rowsPerPageTitle), findsOneWidget);
  });
```
Good call on adding default test.

This can be cleaned up  a little bit and test the `MaterialLocalizations` string.